renderer.js

```diff
@@ -111,31 +133,27 @@ function getUnsignedApkOptions() {
                     "src": "/favicon.png"
                 }
             ]
-        }],
-        signingMode: "none",
-        signing: null,
+        }],*///sundy 
         splashScreenFadeOutDuration: 300,
-        startUrl: "/saved",
-        themeColor: "#3f51b5",
-        webManifestUrl: "https://sadchonks.com/manifest.json"
+        themeColor: "#3f51b5"
     };
 }
 
-function getSignedApk() {
```

Comment:
Is there a reason you used getUnsignedApkOptions() instead of getSignedApk() when the generated APK is a signed release?